### PR TITLE
3 | csd | Setup initial Qwik

### PR DESCRIPTION
**Pull request by cstroliadavis**
NOTE:
 There may be some lint rules that qwik would recommend to be added that are unique to qwik.
 I'll see if I can look those up and add them at a later date.

For now, things seem to be working, obviously it's just  a blank screen so far.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,7 +103,6 @@ dist
 
 # vuepress v2.x temp and cache directory
 .temp
-.cache
```
Comment (Tekner) [inline .gitignore:105]: Why did you remove .cache?

Comment (cstroliadavis) [reply .gitignore:105]: because there were 2 in the file

Comment (Tekner) [reply .gitignore:105]: .cache and .cache/ are not equivalent. 
The former matches files or directories with that name, the latter matches only a directory. Not only that, but most entries that appear to be for directories use the former pattern. I think we should be consistent.
Personally, I am in favor of explicitly having directories with trailing slashes. Furthermore, I am also in favor of explicitly leading directories with a slash if they are only going to be in the root, if only for readability's sake. Thoughts?

Comment (cstroliadavis) [reply .gitignore:105]: Indeed. There is still a .cache on line 84. I removed the duplicate on line 105.

Comment (Tekner) [reply .gitignore:105]: Oh I missed that one. So there is still a duplicate, then because line 84 covers line 96

File: src/global.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  line-height: inherit;
+}
+
+/**
```
[the remark this answers is not in the document]
Comment (cstroliadavis) [reply src/global.css:34]: hmm. did not notice that. I'll fix.

This was basically the file created by `pnpm create qwik`. I did do some minor modifications to remove experimental versions for various browsers that have had the full real version of those particular CSS items for years now. (FYI: -webkit-text-size-adjust does not exist on all browsers, yet as close as I can tell)

**Review by celeron450 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (Tekner) [reply .gitignore:105]: Yes it does. That's what I was saying in my first response message. Lack of trailing slash is interpreted as ambiguous and will match either a file _or_ a directory. This is why I'm suggesting we be consistent and always include a slash if it's a directory and also lead it with a slash if it's in the root.
